cmd/set/theme/theme.go

```diff
@@ -0,0 +1,246 @@
+package theme
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+	"github.com/spf13/cobra"
+)
+
+var theme string
+var option string
+var ListTheme = []string{"London", "Tokyo", "Helsinki"}
+
+type body struct {
+	PageType string     `json:"PageType"`
+	CustomJS []CustomJS `json:"CustomJS"`
+}
+type CustomJS struct {
+	Content  string `json:"Content"`
+	fileName string `json:"fileName"`
+}
+
+func NewThemeCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "theme",
+		Short: "Changes the theme of the site",
+		Long: heredoc.Doc(`
+		This command changes the theme of the site depending on the user's choice.
+		`),
+		Example: heredoc.Doc(`
+			$ lr set theme --theme <theme>
+			
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if theme == "" {
+				return &cmdutil.FlagError{Err: errors.New("`theme` is required argument")}
+			}
+			valid := contains(ListTheme, theme)
+			if !valid {
+				return &cmdutil.FlagError{Err: errors.New("Please Enter a valid theme")}
+			}
+			return themes()
+		},
+	}
+	fl := cmd.Flags()
+	fl.StringVarP(&theme, "theme", "t", "", "Changes the theme")
+
+	return cmd
+}
+
+func themes() error {
+	currentTheme, err := getTheme()
+	if err != nil {
+		return err
+	}
+	if theme == currentTheme {
+		fmt.Println("You are already using this theme")
+		return nil
+	} else {
+		fmt.Printf("Previous changes will be lost. Press Y to continue: ")
+		fmt.Scanf("%s", &option)
+		if option != "Y" {
+			return nil
+		}
+	}
+
+	err = resetCalls()
+	if err != nil {
+		return err
+	}
+
+	err = themeurl()
+	if err != nil {
+		return err
+	}
+
+	err = hostedPageCalls()
+	if err != nil {
+		return err
+	}
+
+	err = otherCalls()
+	if err != nil {
+		return err
+	}
+
+	err = updateCalls()
+	if err != nil {
+		return err
+	}
+
```

Comment:
As this process is time consuming we can provide the message after each api calls something like this
```
fmt.printf("Reseting Your current theme...")
err = resetCalls()
	if err != nil {
		return err
	}
fmt.printf("Updating the CSS based on new theme.." )
err = themeurl()
	if err != nil {
		return err
	}
...
...
...
```